{"uuid":"1184aa5f-026b-43dd-95c1-e70d9587fb89","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["1c8f93c1-63f8-4d60-9e23-42067c287497"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687895941589,"stop":1687895941589},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687895941826,"stop":1687895941826},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687895941949,"stop":1687895941949},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687895942000,"stop":1687895942000},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687895942019,"stop":1687895942019}],"start":1687895941469,"stop":1687895942019}